src/App.tsx

```diff
@@ -220,6 +219,7 @@ function App() {
   });
 
   const hasDust = useAppSelector(state => {
```

Comment:
We can pull in this flag into the component as well and condense the multiple if statements in here into one

Reply:
Fixed 👍